pkgs/applications/misc/mepo/default.nix

```diff
@@ -8,22 +8,32 @@
 , SDL2_gfx
 , SDL2_image
 , SDL2_ttf
+, jq
+, ncurses
+, inconsolata-nerdfont
+, dmenu
+, xdotool
+, bemenu
+, withWayland ? false
 }:
 
 stdenv.mkDerivation rec {
   pname = "mepo";
-  version = "0.3";
+  version = "0.4.1";
 
   src = fetchFromSourcehut {
     owner = "~mil";
     repo = pname;
     rev = version;
-    hash = "sha256-B7BOAFhiOTILUdzh49hTMrNNHZpCNRDLW2uekXyptqQ=";
+    hash = "sha256-soIAQgo8Cyii/zzJapgJmC5hN/ySnHKoSbOg/XgTnGg=";
   };
 
   nativeBuildInputs = [ pkg-config zig ];
 
-  buildInputs = [ curl SDL2 SDL2_gfx SDL2_image SDL2_ttf ];
+  buildInputs = [
+    curl SDL2 SDL2_gfx SDL2_image SDL2_ttf # linked libraries
+    jq ncurses inconsolata-nerdfont # scripts' dependencies
```

Comment:
I guess it will not work, just check `result/bin/mepo_dl.sh`, there is no `jq` in PATH.

Reply:
I added program dependencies to propagatedBuildInputs.  It sounds wrong but it works.

Reply:
If I got it right, `propagatedBuildInputs` doesn't help if one uses mepo without installing it.

Reply:
I tested with `NIX_PATH=.. nix-shell -p mepo --run mepo` and e.g. dmenu is available but it wasn't before.

Reply:
But if run something like `/nix/store/...-mepo-0.4.1/bin/mepo_dl.sh` without nix-shell or nix-env, `jq` will not be available.

Reply:
You want to use makeWrapper instead of propagatedBuildInputs.

Reply:
Could you elaborate on how makeWrapper should be used, @SuperSandro2000?  I haven't been able to figure that out.

Reply:
Something like this:
```nix
wrapProgram $out/bin/mepo_dl.sh \
  --prefix PATH : ${lib.makeBinPath [ jq ]}
```

Reply:
Thanks, it should work now.  [Sorry 4 the wait](https://www.youtube.com/watch?v=e_fZbUpGplA).

Reply:
Hmmm I think I should wrap scripts explicitly, and either way one has to add `/nix/store/...-mepo-0.4.1/bin` to PATH.

Reply:
Just add `--prefix PATH : $out/bin`.

Reply:
Thanks!

Reply:
Please use suffix. If there is already a jq in path we should probably take it instead.